src/app/content/constants.ts

```diff
@@ -9,6 +9,7 @@ export const maxResourcesPerFetch = 10;
 export const summaryPageSize = 20;
 export const loadMoreDistanceFromBottom = 50;
 
+export const messageQueryParameterName = 'message';
```

Comment:
@TomWoodward you deleted this before - did you mean to move it elsewhere?

Reply:
no i didn't mean to delete it